### ISSUE TITLE
Having added further line(s) to "extra keys view", all lines are shown "compressed" til restart

### ISSUE THREAD

**Issue by mglh** [termux/termux-app#1314]
For me, having added further line(s) to the default "extra keys view" with 
```extra-keys = [ [ ... ] , [ ... ] , [ ... ] ]``` 
in "~/.termux/termux.properties", and having run 
```termux-reload-settings```, 
at first all the lines are shown "compressed";
the chars / signs in them are shown cropped at top and bottom.

Only after the session / the app is left (with ```exit``` or CTRL-D) and started again, 
the lines are shown with the correct heigh.

---

```
$ termux-info
Updatable packages:
All packages up to date
Subscribed repositories: https://dl.bintray.com/grimler/science-packages-24 science/stable
https://dl.bintray.com/grimler/game-packages-24 games/stable
https://dl.bintray.com/termux/termux-packages-24 stable/main
System information:
Linux localhost 3.4.113-g75e7c47 #1 SMP PREEMPT Sat Jan 13 09:11:09 CET 2018 armv7l Android
Termux-packages arch:
arm Android version:
7.1.2
Device manufacturer: motorola Device model:
Moto G 2014
$
```

**Comment by xeffyr** [comment]
Seems to be a duplicate of https://github.com/termux/termux-app/issues/1003.
